load-files.php

```diff
@@ -17,9 +17,7 @@
 function wpephpcompat_load_files() {
 	require_once dirname( __FILE__ ) . '/src/wpephpcompat.php';
 
-	if ( version_compare( phpversion(), '5.3', '<' ) ) {
-		$autoload_file = dirname( __FILE__ ) . '/php52/vendor/autoload_52.php';
-	} else {
+	if ( version_compare( phpversion(), '5.2', '>' ) ) {
```

Comment:
There really isn't a reason to have the `version_compare` clause here. We can just autoload the file.

```php
require_once dirname( __FILE__ ) . '/vendor/autoload.php';
```